coverage_detail/gcov.css

```
@@ -0,0 +1,1125 @@
+
+/* Source code view: line number format */
+span.lineNum
+{
+  background-color: #efe383;
+}
+
+/* Source code view: line number format when there are deleted
+   lines in the corresponding location */
+span.lineNumWithDelete
+{
+  foreground-color: #efe383;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)